notebooks/data-sources/gcsweb-ci/build-logs/model_seldon.ipynb

```diff
@@ -0,0 +1,562 @@
+{
```

Comment:
<p>Any chance we can hand label these 25 to provide some performance metrics here as well? Also, map the outputs to the correct human readable label "pass, infra failure, application failure" (I think)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Maybe i'm being <em>extra</em> cautious but is it possible to sanity check that the results from the endpoint exactly match the results that we'd get if <code>test_list</code> was passed to <code>pipeline.predict</code> here in the notebook? Just to make sure seldon is using the correct model and its behaving as expected :)</p>

Reply:
@chauhankaranraj the test sets are now same.

@MichaelClifford good idea! I'll create an issue to create a test set and label it, I'll add Pep to it as well since he knows the data in and out.